raiden/ui/app.py

```diff
@@ -267,15 +267,24 @@ def run_app(
 
     config['chain_id'] = given_network_id
 
-    log.debug('Network type', type=network_type)
-    if network_type == 'main':
-        config['network_type'] = NetworkType.MAIN
+    if environment == Environment.DEVELOPMENT and network_id == 1:
+        click.secho(
+            f'Using environment setting to "production" as the chosen network '
+            f'{ID_TO_NETWORKNAME[node_network_id]} has no test configuration.',
+            fg='yellow',
+        )
+        environment = Environment.PRODUCTION
```

Comment:
The `sys.exit(1)` was removed, is it intentional?

Reply:
Yes. The point is just to not run with the `development`/`test` configuration if we are on mainnet. So it's enough to override that setting, instead of aborting the whole thing.

Reply:
No @jomuel please make it abort. It's much more explicit this way to the user that he provided an illegal combindation of arguments.

Reply:
For the release these should be the default arguments, so he should provide nothing specific.

Reply:
I see. I dropped that commit and made a new one with just an updated and more helpful error message instead.